modules/apps/layout/layout-page-template-api/src/main/resources/com/liferay/layout/page/template/validator/dependencies/page_definition_json_schema.json

```diff
@@ -731,6 +731,37 @@
 				"layout": {
```

Comment:
I think that this "SectionDefinition" should be named "ContainerDefinition", but @jkappler told me that would be a breaking change in the API :shrug: 
Moreover, nesting some fields inside layout probably doesn't make sense anymore, because now, in the markup, we have an unique "div" tag with all properties.